actions/commitizen-cz-cli/plugin.yaml

```diff
@@ -0,0 +1,24 @@
+version: 0.1
+actions:
+  definitions:
+    - id: commitizen-cz-cli
+      display_name: Commitizen (cz-cli)
+      description: Assistant for writing commit messages that obey a standard
+      runtime: node
+      packages_file: package.json
```

Comment:
I've opened an internal feature request to support tool dependencies in actions. It would be nice not to need the `package_file` here.

Reply:
Sounds good!